app/assets/stylesheets/omni_nav/modules/omni-nav.scss

```diff
@@ -976,11 +1009,11 @@ a.cu_nav_button {
 }
 
 .cu-search-open-trigger {
+    z-index: 99999;
```

Comment:
I assume this value is connected with previous one. Add comment here noting that.